src/components/apps/launch/AppInfo.js

```diff
@@ -97,6 +108,27 @@ const UnavailableMsg = ({ app, hasDeprecatedParams, baseId }) => {
                 }}
             />
         );
+    } else if (computeLimitExceeded) {
+        message = (
+            <Trans
+                t={t}
+                i18nKey="computeLimitExceeded"
+                components={{
+                    buy: (
+                        <Link
```

Comment:
I haven't been able to get that to work. When I try to use `ExternalLink`, I keep getting this error in Storybook:

```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

Is there anything that I'm missing that might fix that error while still using `ExternalLink`?

Reply:
I'd probably have to see your changes, but did you mix up `import { ExternalLink } from "components/utils/ExternalLink";` instead of `import ExternalLink from "components/utils/ExternalLink";` as in the second suggestion for the error?

Here's an example of how it's used in a `Trans` component like this:
https://github.com/cyverse-de/sonora/blob/6d19ea18a979979d7b942485c2c05b5a097e95b0/src/components/utils/ComingSoonInfo.js#L32-L40

So here `onClick` can be replaced with `href={config?.subscriptions?.checkout_url}`.

Reply:
My mistake. I'd forgotten that the input was different. I'll give that a try. Thanks!

Reply:
s/input/import/